src/common/backtrace.c

```diff
@@ -0,0 +1,206 @@
+#include <execinfo.h>
+#include <stdio.h>
+#include <signal.h>
+#include <assert.h>
+#include <signal.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <err.h>
+ 
+void cause_segfault();
+void cause_crash();
+ 
+static char const * _program_name = "mme_app";
```

Comment:
program name seems fixed? is it updated for other apps ??

Reply:
Fixed.